src/Microsoft.ML.Data/Scorers/BinaryClassifierScorer.cs

```diff
@@ -199,23 +199,26 @@ private protected override void SaveAsOnnxCore(OnnxContext ctx)
             for (int iinfo = 0; iinfo < Bindings.InfoCount; ++iinfo)
                 outColumnNames[iinfo] = Bindings.GetColumnName(Bindings.MapIinfoToCol(iinfo));
 
-            string opType = "Binarizer";
+            string scoreColumn = Bindings.RowMapper.OutputSchema[Bindings.ScoreColumnIndex].Name;
+
             OnnxNode node;
-            var binarizerOutput = ctx.AddIntermediateVariable(null, "BinarizerOutput", true);
+            string opType = "Binarizer";
+            var binarizerOutput = ctx.AddIntermediateVariable(NumberDataViewType.Single, "BinarizerOutput", false);
```

Comment:
>false [](start = 108, length = 5)

What does this false mean?

Reply:
False = Do not skip adding shape and type information (or in other words, add shape and type information). 

---
In reply to: [383088227](https://github.com/dotnet/machinelearning/pull/4878#discussion_r383088227) [](ancestors = 383088227)